charts/home-assistant/0.0.1/questions.yaml

```diff
@@ -0,0 +1,324 @@
+                schema:
+                  type: path
+                  default: "/config"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "Config - Enable Host Path"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Config Host Path"
+                      schema:
+                        type: hostpath
+                        required: true
+        # Z-Wave device -----------------
+        - variable: zwave
+          label: "Z-Wave Device"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "USB Z-Wave Device"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Path to Z-Wave device on host"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyAMC0"
+                    - variable: mountPath
+                      label: "Path to mount device inside the container"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyAMC0"
+        # Zigbee device -----------------
+        - variable: zigbee
+          label: "Zigbee Device"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "USB Zigbee Device"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Path to Zigbee device on host"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyAMC1"
+                    - variable: mountPath
+                      label: "Path to mount device inside the container"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyAMC1"
+  # Enable privileged
+  - variable: securityContext
+    group: "Storage"
+    label: "Security Context"
+    schema:
+      type: dict
+      attrs:
+        - variable: privileged
+          label: "Enable privileged to pass thru a USB device from the host"
+          schema:
+            type: boolean
+            default: false
+
+  # Enable Host Network
+  - variable: hostNetwork
+    group: "Networking"
+    label: "Enable Host Network"
```

Comment:
```suggestion
    label: "Enable Host Network"
    description: "Required if you want devices to be discoverable by Home Assistant"
```

Reply:
This is also  a MAJOR security flaw, its much more nuanced than just saying "required".

I don't know how (if) this works with services and ingress.
It would also requires `dnsPolicy: ClusterFirstWithHostNet` or it would lose the ability to communicate with any other pods

As such I would like good testing on the consequences for other features pro-and-con before merge. Or significant rephrasing of this description.

Reply:
Okey, I am going to merge it, but it needs significant rework in the questions.yaml to correctly set the dnsPolicy.
It also needs testing with Ingress.